ethminer/MinerAux.h

```diff
@@ -219,6 +245,18 @@ class MinerCLI
 			m_minerType = MinerType::CPU;
 		else if (arg == "-G" || arg == "--opencl")
 			m_minerType = MinerType::GPU;
+		else if (arg == "-U" || arg == "--cuda") {
+			m_minerType = MinerType::CUDA;
+
+			cout << "Genoil's CUDA ethminer" << endl;
+			cout << "=====================================================================" << endl;
+			cout << "Forked from github.com/ethereum/cpp-ethereum" << endl;
+			cout << "Ported from tim hughes' OpenCL kernel" << endl;
+			cout << "With contributions from tpruvot and sp_ " << endl << endl;
+			cout << "Please consider donating a tiny fraction of the extra performance to:" << endl;
+			cout << "ETH: 0xb9310b185455f863f526dab3d245809f6854b4d" << endl;
```

Comment:
oh crap i'm so bad at git. the txt isn't supposed to in the PR...but what did you mean anyway?